arionBackend/tests/test_views.py

```diff
@@ -15,49 +14,9 @@ class ViewsTestCase(TestCase):
 
     @classmethod
     def setUpTestData(cls):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename method "setUpTestData" to match the regular expression ^[a-z_][a-z0-9_]{2,}$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS100)